lib/utils/form.ts

```diff
@@ -65,24 +63,20 @@ export function checkEligible(application: Application): [boolean, string[]] {
     'home-how-many-bedrooms'
   );
 
-  if (
-    bedroomNeed <= requestedNumberOfBedrooms &&
-    numberOfApplicantsWithMedicalNeeds === 0
-  ) {
-    setInvalid(disqualificationReasonOptions.notLackingRooms);
+  if (bedroomNeed <= requestedNumberOfBedrooms) {
+    setInvalid('notLackingRooms');
   }
-  //******** *//
 
   if (!isOver18(mainApplicant)) {
-    setInvalid(disqualificationReasonOptions.under18YearsOld);
+    setInvalid('under18YearsOld');
   }
 
-  for (const [form, values] of Object.entries(FormID)) {
-    const eligibilityCriteria = getEligibilityCriteria(values);
+  for (const [, formID] of Object.entries(FormID)) {
```

Comment:
I realise this is some legacy code not part of this PR, but if we only need the value from the enum maybe we use Object.values instead of Object.entries:
```suggestion
  for (const formID of Object.values(FormID)) {
```